# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR ORGANIZATION
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"POT-Creation-Date: 2019-05-08 16:46-0300\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=cp1252\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: pygettext.py 1.5\n"


#: console.py:97
msgid "UNHANDLED EXCEPTION in console.py. This should not happen."
msgstr ""

#: exception_handling.py:23
msgid ""
"    AvantPy exception: {exception_name}\n"
"\n"
"    Error found in file '{filename}' on line {linenumber}.\n"
"\n"
"{partial_source}\n"
"\n"
msgstr ""

#: exception_handling.py:32
msgid ""
"    Python exception: \n"
"        {python_display}\n"
"\n"
"    Error found in file '{filename}' on line {linenumber}.\n"
"\n"
"{partial_source}\n"
"\n"
msgstr ""

#: exception_handling.py:122
msgid ""
"An exception was raised for which we have no simplified traceback:\n"
msgstr ""

#: exception_handling.py:165
msgid ""
"    The AvantPy '{nobreak_kwd}' keyword cannot be used in\n"
"    an '{if_kwd}/{elif_kwd}/{else_kwd}' clause (Python: if/elif/else).\n"
"\n"
msgstr ""

#: exception_handling.py:189
msgid ""
"    In this case, the line indicated by an arrow\n"
"    is more indented than expected and does not match\n"
"    the indentation of the previous line.\n"
msgstr ""

#: exception_handling.py:195
msgid ""
"    In this case, the line indicated by an arrow\n"
"    was expected to begin a new indented block.\n"
msgstr ""

#: exception_handling.py:200
msgid ""
"    In this case, the line indicated by an arrow\n"
"    is less indented the preceding one,\n"
"    and is not aligned vertically with another block of code.\n"
msgstr ""

#: exception_handling.py:220
msgid ""
"    An indentation error occurs when a given line is\n"
"    not indented (aligned vertically) as expected.\n"
"{this_case}\n"
msgstr ""

#: exception_handling.py:252
msgid ""
"    The opening {open_bracket} does not match the closing {close_bracket}.\n"
"\n"
msgstr ""

#: exception_handling.py:283
msgid ""
"    The closing {bracket} does not match anything.\n"
"\n"
msgstr ""

#: exception_handling.py:309
msgid ""
"    A statement beginning with the '{repeat_kwd}' keyword must be on\n"
"    a single line ending with a colon (:) that indicates the beginning of\n"
"    an indented block of code, with no other colon appearing on that line.\n"
"\n"
msgstr ""

#: exception_handling.py:338
msgid ""
"    The AvantPy '{keyword}'' keyword can be used only when preceded by '{repeat_kwd}'.\n"
"\n"
msgstr ""

#: exception_handling.py:370
msgid ""
"    A NameError exception indicates that a variable or\n"
"    function name is not known to Python.\n"
"    Most often, this is because there is a spelling mistake; however,\n"
"    sometimes it is because it is used before being defined\n"
"    or given a value. In your program, the unknown name is '{var_name}'.\n"
"\n"
msgstr ""

#: exception_handling.py:403
msgid ""
"    The AvantPy '{nobreak_kwd}' keyword can be used instead of '{else_kwd}'\n"
"    (Python: else) only when it begins a new statement in\n"
"    '{for_kwd}/{while_kwd}' loops (Python: for/while).\n"
"\n"
msgstr ""

#: exception_handling.py:434
msgid ""
"    The AvantPy '{nobreak_kwd}' keyword can only be used as a replacement\n"
"    of '{else_kwd}' (Python: else) with a matching '{for_kwd}' or\n"
"    '{while_kwd}' loop (Python: for/while).\n"
"\n"
msgstr ""

#: exception_handling.py:465
msgid ""
"    The AvantPy '{repeat_kwd}' keyword can only be used to begin\n"
"    a new loop (Python: equivalent to 'for' or 'while' loop).\n"
"\n"
msgstr ""

#: exception_handling.py:498
msgid ""
"    A TabError indicates that you have used both spaces\n"
"    and tab characters to indent your code.\n"
"    This is not allowed in Python.\n"
"    Python's recommendation is to always use spaces to indent your code.\n"
"\n"
msgstr ""

#: exception_handling.py:527
msgid ""
"    The AvantPy '{nobreak_kwd}' keyword cannot be used in\n"
"    a '{try_kwd}/{except_kwd}/{else_kwd}/{finally_kwd}' clause\n"
"    (Python: try/except/else/finally).\n"
"\n"
msgstr ""

#: exception_handling.py:554
msgid ""
"    AvantPy exception: UnknownDialectError\n"
"\n"
"    The following unknown dialect was requested: {dialect}.\n"
"\n"
"    The known dialects are: {all_dialects}.\n"
"\n"
msgstr ""

#: exception_handling.py:573
msgid ""
"    AvantPy exception: UnknownLanguageError\n"
"\n"
"    The following unknown language was requested: {lang}.\n"
"\n"
"    The known languages are: {all_langs}.\n"
"\n"
msgstr ""

#: exceptions.py:14
msgid "AvantPy exception:"
msgstr ""

#: exceptions.py:30
msgid "AvantPy syntax error:"
msgstr ""

#: exceptions.py:44
msgid ""
"The AvantPy '{nobreak_kwd}' keyword cannot be used in\n"
"an '{if_kwd}/{elif_kwd}/{else_kwd}' clause (Python: if/elif/else).\n"
msgstr ""

#: exceptions.py:66
msgid ""
"The opening {open_bracket} does not match the closing {close_bracket}.\n"
msgstr ""

#: exceptions.py:86
msgid ""
"The closing {bracket} does not match anything.\n"
msgstr ""

#: exceptions.py:97
msgid ""
"A statement beginning with the '{repeat_kwd}' keyword must be on\n"
"a single line ending with a colon (:) that indicates the beginning of\n"
"an indented block of code, with no other colon appearing on that line.\n"
msgstr ""

#: exceptions.py:111
msgid ""
"The AvantPy '{keyword}'' keyword can be used only whenpreceded by '{repeat_kwd}'.\n"
msgstr ""

#: exceptions.py:122
msgid ""
"The AvantPy '{nobreak_kwd}' keyword can be used instead of '{else_kwd}'\n"
"(Python: else) only when it begins a new statement in\n"
"'{for_kwd}/{while_kwd}' loops (Python: for/while).\n"
"\n"
msgstr ""

#: exceptions.py:139
msgid ""
"The AvantPy '{nobreak_kwd}' keyword can only be used as a replacement\n"
"of '{else_kwd}' (Python: else) with a matching '{for_kwd}' or\n"
"'{while_kwd}' loop (Python: for/while).\n"
msgstr ""

#: exceptions.py:156
msgid ""
"The AvantPy '{repeat_kwd}' keyword can only be used to begin\n"
"a new loop (Python: equivalent to 'for' or 'while' loop).\n"
msgstr ""

#: exceptions.py:167
msgid ""
"The AvantPy '{nobreak_kwd}' keyword cannot be used in\n"
"a '{try_kwd}/{except_kwd}/{else_kwd}/{finally_kwd}' clause\n"
"(Python: try/except/else/finally).\n"
msgstr ""

#: exceptions.py:188
msgid ""
"The following unknown dialect was requested: {dialect}.\n"
"\n"
"The known dialects are: {all_dialects}.\n"
msgstr ""

#: exceptions.py:205
msgid ""
"The following unknown languages was requested: {lang}.\n"
"\n"
"The known dialects are: {all_langs}.\n"
msgstr ""

#: import_hook.py:40
msgid "Cannot find main module: "
msgstr ""

#: session.py:158
msgid "    ==> Language: {lang} | AvantPy dialect: {dialect}"
msgstr ""

